bot/exts/moderation/infraction/infractions.py

```diff
@@ -319,6 +356,28 @@ async def apply_ban(
         bb_reason = "User has been permanently banned from the server. Automatically removed."
         await bb_cog.apply_unwatch(ctx, user, bb_reason, send_message=False)
 
+    @respect_role_hierarchy(member_arg=2)
+    async def apply_voice_ban(self, ctx: Context, user: UserSnowflake, reason: t.Optional[str], **kwargs) -> None:
+        """Apply a voice ban infraction with kwargs passed to `post_infraction`."""
+        if constants.Roles.voice_verified not in [role.id for role in user.roles]:
+            await ctx.send(":x: Can't apply Voice Ban to user who have not passed the Voice Gate.")
+            return
+
```

Comment:
This seems like an arbitrary restriction, there is no reason why we can't voice ban users who have not passed the gate.
```suggestion
```